admin/templates/configuration/tmp/configure/sql/qserv-worker.sql

```diff
@@ -1,6 +1,7 @@
--- Used by xrootd Qserv plugin:
+CREATE USER IF NOT EXISTS 'qsmaster'@'localhost';
```

Comment:
Why not use "CREATE USER IF NOT EXISTS" in both cases (or DROP USER if you want to get rid of account)? I understand that "OR REPLACE" is a relatively new mariadb extension, we should try to avoid depending on mariadb-specific features for now.